src/components/videoPlayer/videoPlayerView.tsx

```diff
@@ -1,37 +1,35 @@
-import React, { useState } from 'react';
+import React, { useState, useRef } from 'react';
 import { Dimensions } from 'react-native';
 import { View, StyleSheet, ActivityIndicator } from 'react-native';
 import WebView from 'react-native-webview';
 import YoutubeIframe, { InitialPlayerParams } from 'react-native-youtube-iframe';
-import { WebViewSource } from 'react-native-webview/lib/WebViewTypes';
+import Video from 'react-native-video';
 
 interface VideoPlayerProps {
-  mode: 'source'|'youtube'|'url';
+  mode: 'source' | 'youtube';
   contentWidth?: number;
   youtubeVideoId?: string;
-  videoUrl?: string;
   startTime?: number;
-  source?: WebViewSource;
-
+  source?: string;
```

Comment:
let's rename `source` to `uri`, as source prop name can be misleading.

Reply:
and same goes for available values of `mode` prop

Reply:
@noumantahir updated the props.